domains/Accounts/Database/Migrations/2020_10_04_140743_create_users_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('users', function (Blueprint $table) {
+            $table->id();
+            $table->string('name')->index();
+            $table->string('email')->unique();
+            $table->string('password');
+            $table->boolean('trusted')->default(false);
+            $table->timestamp('email_verified_at')->nullable();
```

Comment:
I know the original issue reference that we wanted timestamp columns without timezone. However, to future proof this a little, we changed the issue description to have that information.

Care to add the timezone definition to this table columns?

Reply:
OK. Done so for created/updated/deleted_at columns as well.